clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -147,7 +143,7 @@ class FossIdClientNewProjectTest : StringSpec({
 
     "A scan can be deleted" {
         service.deleteScan("", "", SCAN_CODE) shouldNotBeNull {
-            checkResponse("delete scan", true)
+            checkResponse("delete scan")
```

Comment:
Not: Ideally, the (somewhat unrelated) omission of the default argument would ideally go to a separate / preceding commit.